docs/app/pages/components/sections/date-time-picker/time-picker/time-picker.component.less

```diff
@@ -0,0 +1,18 @@
+timepicker a .glyphicon {
```

Comment:
This less needs to be part of the library, as I explained before. Could you remove this from the documentation component and add it into src/styles?